sqlmesh/core/engine_adapter/bigquery.py

```diff
@@ -182,7 +182,32 @@ def query_factory() -> Query:
     def _begin_session(self, properties: SessionProperties) -> None:
         from google.cloud.bigquery import QueryJobConfig
 
-        job = self.client.query("SELECT 1;", job_config=QueryJobConfig(create_session=True))
+        query_label = []
+        if "query_label" in properties and isinstance(properties["query_label"], exp.Array):
+            for label_tuple in properties["query_label"].expressions:
+                if (
+                    isinstance(label_tuple, exp.Tuple)
+                    and len(label_tuple.expressions) == 2
+                    and all(isinstance(label, exp.Literal) for label in label_tuple.expressions)
```

Comment:
what if it's an identifier and not a string literal eg: `query_labels = [ label_a ]` and not `query_labels = [ 'label_a' ]`. Might worth support both approaches.

Reply:
We could simply require strings; not sure what's the value of supporting identifiers here. Seems like an unnecessary increase in scope?

Reply:
Sounds good. I'm good with that